app/src/main/java/cm/aptoide/pt/app/view/AppViewPresenter.java

```diff
@@ -119,31 +127,18 @@ public AppViewPresenter(AppViewView view, AccountNavigator accountNavigator,
     resumeDownload();
     cancelDownload();
     handleApkfyDialogPositiveClick();
-
     handleDismissWalletPromotion();
-    handleInstallWalletPromotion();
```

Comment:
This was removed, and now added only for eskills apps. Is this the intended behavior ?

Reply:
added it back, also added a filter inside handleInstallWalletPromotion, please check